File: PpeGSBTeam27/PpeGSBTeam27/Produit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PpeGSBTeam27
{
    class Produit
    {
        int num;
        String nom;
        String effet;
        String contreIndication;
        String presentation;
        String dosage;
        String famille;
        String interaction;
        double prixHT;
        double prixEchantillon;

        public Produit(int unNum, string unNom, string unEffet, string desCI, string unePresentation, string unDosage, int unIDFamille, double unPrixHT, double unPrixEch, string uneInteraction)
        {
            num = unNum;
            nom = unNom;
            effet = unEffet;
            contreIndication = desCI;
            presentation = unePresentation;
            dosage = unDosage;
            famille = ;
            prixHT = unPrixHT;
            prixEchantillon = unPrixEch;
            interaction = uneInteraction;
        }

        #region Getters et Setters

        public int getNum()
        {
            return num;
        }

        public string getNom()
        {
            return nom;
        }

        public void setNom(string value)
        {
            this.nom = value;
        }


        public void setEffet(string value)
        {
            this.effet = value;
        }

        public string getEffet()
        {
            return effet;
        }

        public void setContreIndication(string value)
        {
            this.contreIndication = value;
        }

        public string getContreIndication()
        {
            return contreIndication;
        }

        public void setPresentation(string value)
        {
            this.presentation = value;
        }

        public string getPresentation()
        {
            return presentation;
        }

        public void setDosage(string value)
        {
            this.dosage = value;
        }

        public string getDosage()
        {
            return dosage;
        }

        public void setFamille(string value)
        {
            this.famille = value;
        }

        public string getFamille()
        {
            return famille;
        }

        public void setPrixHT(double value)
        {
            this.prixHT = value;
        }

        public double getPrixHT()
        {
            return this.prixHT;
        }

        public void setPrixEchantillon(double value)
        {
            this.prixEchantillon = value;
        }

        public double getPrixEchantillon()
        {
            return prixEchantillon;
        }

        public string getInteraction()
        {
            return interaction;
        }
        #endregion

    }
}

File: PHP/PPE_PHP/src/Domain/Produit.php
<?php

namespace PPE_PHP\Domain;

class Produit 
{
    /**
     * Produit id_produit.
     *
     * @var integer
     */
    private $id_produit;

    /**
     * Produit id_famille.
     *
     * @var integer
     */
    private $id_famille;

    /**
     * Produit nom.
     *
     * @var string
     */

    private $nom;

    /**
     * Produit effets.
     *
     * @var string
     */
    private $effets;

    /**
     * Produit contre_indication.
     *
     * @var string
     */
    private $contre_indication;

    /**
     * Produit interactions_autres_produits.
     *
     * @var string
     */
    private $interactions_autres_produits;
    /**
     * Produit presentation.
     *
     * @var string
     */
    private $presentation;

    /**
     * Produit dosage.
     *
     * @var double
     */
    private $dosage;

    /**
     * Produit prix_HT.
     *
     * @var double
     */
    private $prix_HT;

    /**
     * Produit prix_Echantillon.
     *
     * @var double
     */
    private $prix_Echantillon;

    // id_produit
    public function getIdProduit() {
        return $this->id_produit;
    }

    public function setIdProduit($id_produit) {
        $this->id_produit = $id_produit;
    }

    // id_famille
    public function getIdFamille() {
        return $this->id_famille;
    }

    public function setIdFamille($id_famille) {
        $this->id_famille = $id_famille;
    }

    // nom
    public function getNom() {
        return $this->nom;
    }

    public function setNom($nom) {
        $this->nom = $nom;
    }

    // effets
    public function getEffets() {
        return $this->effets;
    }

    public function setEffets($effets) {
        $this->effets = $effets;
    }

    // contre_indication
    public function getContreIndication() {
        return $this->contre_indication;
    }

    public function setContreIndication($contre_indication) {
        $this->contre_indication = $contre_indication;
    }



    private $contre_indication;
private $interactions_autres_produits;
private $presentation;
private $dosage;
private $prix_HT;
$prix_Echantillon;
}
